### PR TITLE
Add ability to asynchronously reset the cache

### PR DESCRIPTION
**Pull request by Napolskih**
https://redis.io/commands/flushdb
https://github.com/redis/redis-rb/pull/706

For Redis 4.0.0 or greater and redis-rb 4 or greather.

`Rails.cache.clear(async: true)`

### CODE REVIEW

File: lib/readthis/cache.rb [hunk 1]
```diff
@@ -374,8 +374,14 @@ def exist?(key, options = {})
     #
     #   cache.clear #=> 'OK'
     #
-    def clear(_options = nil)
-      invoke(:clear, '*', &:flushdb)
+    def clear(options = nil)
```
Comment (sorentwo) [inline lib/readthis/cache.rb]: This can be changed to `options = {}`, and then you can simply do `if options[:async]` within the body.

**Comment by Napolskih** [comment]
🆙

**Comment by Napolskih** [comment]
hi
I do not see this commit in master 😄

**Comment by sorentwo** [comment]
@Napolskih It must have been force pushed over, sorry. It was in my ref-log locally though, so I've cherry-picked it and pushed it back up.

**Pull request merged**